.circleci/config.yml

```diff
@@ -253,6 +256,556 @@ commands:
       #     name: Push maintenance application
       #     command: |
       #       cd maintenance_page && cf push -s cflinuxfs4 --vars-file ../<<parameters.deploy_config_file >>
+  cf_backup:
+    description: "Login to cloud foundry space with service account credentials, Connect to DB & S3, backup DB to S3"
+    parameters:
+      app_name:
+        description: "Name of Cloud Foundry cloud.gov application; must match
+          application name specified in manifest"
+        type: string
+      auth_client_id:
+        description: "Name of CircleCi project environment variable that
+          holds authentication client id, a required application variable"
+        type: env_var_name
+      auth_client_secret:
+        description: "Name of CircleCi project environment variable that
+          holds authentication client secret, a required application variable"
+        type: env_var_name
+      cloudgov_username:
+        description: "Name of CircleCi project environment variable that
+          holds deployer username for cloudgov space"
+        type: env_var_name
+      cloudgov_password:
+        description: "Name of CircleCi project environment variable that
+          holds deployer password for cloudgov space"
+        type: env_var_name
+      cloudgov_space:
+        description: "Name of CircleCi project environment variable that
+          holds name of cloudgov space to target for application deployment"
+        type: env_var_name
+    steps:
+      - run:
+          name: Install Dependencies
+          command: |
+            # update
+            sudo apt-get update
+            # Install PostgreSQL Client to have access to pg_dump
+            sudo apt-get install -y postgresql-client
+            # Install jq
+            sudo apt-get install -y jq
+            # Install zip & unzip
+            sudo apt-get install -y zip unzip
+            # Install Curl
+            sudo apt-get install -y curl
+            # Install uuid-runtime to have access to uuidgen
+            sudo apt-get install uuid-runtime
+            # Install pv
+            sudo apt-get install pv
+            # Install wget
+            sudo apt-get install wget
+            # Install Cloud Foundry CLI
+            wget -q -O - https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
```

Comment:
why install curl if wget is going to be used?

Reply:
curl is needed to upload the file to s3